app/bundles/LeadBundle/EventListener/ReportSubscriber.php

```diff
@@ -154,7 +163,17 @@ public function onReportBuilder(ReportBuilderEvent $event)
                 ],
             ];
 
-            $leadFields   = $this->fieldModel->getEntities();
+            $leadFields = $this->fieldModel->getEntities([
+                'filter' => [
+                    'force' => [
+                        [
+                            'column' => 'f.object',
+                            'expr'   => 'like',
+                            'value'  => 'lead',
+                        ],
+                    ],
+                ],
+            ]);
             $fieldColumns = [];
             foreach ($leadFields as $f) {
```

Comment:
Let's move this in to a protected method and use it here and on line 314 rather than duplicating the same thing twice.